tests/e2e/matchers/to-have-adsense-tag.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * WordPress dependencies
+ */
+import { createURL } from '@wordpress/e2e-test-utils';
+
+/**
+ * Jest matcher for asserting the URL at the given path contains an Adsense tag.
+ *
+ * @param {string} path The URL path of the current site to check.
+ */
+export async function toHaveAdsenseTag( path ) {
```

Comment:
Nice work with the custom matcher 👍 

Might actually nice to complement this with matchers for GTM and GA tags. I'll have a quick look.

Reply:
@aaemnnosttv What do you think about passing `page` to the matchers instead of the path? For the admin tracking tests we need to look for the tags in the backend, not the frontend:

https://github.com/google/site-kit-wp/blob/82d9f4164e3d400fec8221fa7f750298b17859b7/tests/e2e/specs/admin-tracking.test.js#L47-L85